ext/stackprof/stackprof.c

```diff
@@ -695,12 +738,15 @@ stackprof_signal_handler(int sig, siginfo_t *sinfo, void *ucontext)
 	    _stackprof.unrecorded_gc_sweeping_samples++;
 	}
 	_stackprof.unrecorded_gc_samples++;
-	rb_postponed_job_register_one(0, stackprof_gc_job_handler, (void*)0);
+	rb_postponed_job_register_one(0, stackprof_job_record_gc, (void*)0);
     } else {
 #ifdef USE_POSTPONED_JOB
-	rb_postponed_job_register_one(0, stackprof_job_handler, (void*)0);
+	rb_postponed_job_register_one(0, stackprof_job_sample_and_record, (void*)0);
 #else
-	stackprof_job_handler(0);
+	// Buffer a sample immediately, this may overwrite the existing sample
```

Comment:
Is this comment accurate?  It looks like it returns early if we already have a sample.

Reply:
this is inaccurate. Will fix ❤️

Reply:
done!